src/sentry/api/helpers/group_index.py

```diff
@@ -118,7 +118,7 @@ def validate_search_filter_permissions(organization, search_filters, user):
                     sender=validate_search_filter_permissions,
                 )
                 raise ValidationError(
-                    u'You need access to the advanced search feature to use {}'.format(
+                    u'The {} feature is available for organizations on the new Developer, Team, and Business plans.'.format(
```

Comment:
This is a bit of a sentry/getsentry split thing. On-premise users *could* see this and they have no access to billing plans. Instead they have to modify configuration values. Perhaps @EvanPurkhiser has a suggestion on how to handle this?

Reply:
Ahh crap that's a good point.

Reply:
Yeah this is a little bit tricky since we don't have great tooling in sentry/getsentry. if we could move this error to the frontend and do feature checks there and it would be a bit easier. It also looks like we don't even translate this error.

Are there other errors that could come back from search, or would it just be this one validation error?

Reply:
It looks like yeah, there are other errors that come back from search (that are also untranslated, [here](https://github.com/getsentry/sentry/blob/2fbc441df5ab65ef42f035753c5146e1702a2eed/src/sentry/api/helpers/group_index.py#L85) is an example) but for this I was just trying to squash the problem of mm1 customers hitting it and writing into support confused about what "advanced search" is and why they don't have it / how to get it. I agree moving it to the frontend in getsentry sounds like a better approach.